kafka_consumer/check.py

```diff
@@ -227,8 +227,8 @@ def _validate_consumer_groups(self, val):
                                     assert isinstance(partition, int)
             return val
         except Exception as e:
-            self.log.exception(e)
-            raise Exception("""The `consumer_groups` value must be a mapping of mappings, like this:
+            self.log.exception('')
```

Comment:
> How would you feel about pointing the users to the proper documentation instead? I don't really feel like logs are the place to include configuration instructions.

👍 I vote to drop the logging and just raise a `ValueError` that has a pointer to the docs. Should I just point to `kafka_consumer.conf.example`? That's most likely to be up to date with the actual check...